NitroxClient/GameLogic/InitialSync/PlayerInitialSyncProcessor.cs

```diff
@@ -33,6 +33,11 @@ public PlayerInitialSyncProcessor(Items item, ItemContainers itemContainers, Loc
         AddStep(sync => AddStartingItemsToPlayer(sync.FirstTimeConnecting));
         AddStep(sync => SetPlayerStats(sync.PlayerStatsData));
         AddStep(sync => SetPlayerGameMode(sync.GameMode));
+        AddStep(sync => SetPlayerKeepInventory(sync.KeepInventoryOnDeath));
+    }
+    private void SetPlayerKeepInventory(bool keepInventoryOnDeath)
```

Comment:
I think we should rename all the PlayerKeepInventory to follow the same on death as the property to keep it consistent

Reply:
To me the whole "OnDeath" is an evidence which doesn't need to be mentionned in our code which is why I didn't discuss it

Reply:
I'm happy either way. Just at work we make sure things are very evident to what they are doing at a glance and so this is the kind of thing I would normally do as otherwise people might be thinking what is this keeping inventory for. If people are happy with it being keep inventory I'm happy to stick with it